packages/macro-ai-api-client/src/types/index.ts

```
@@ -0,0 +1,17 @@
+// Unified types index - auto-generated, do not edit manually
+
+// Re-export all domain-specific types
+export * from './auth.types.js'
+export * from './chat.types.js'
+export * from './user.types.js'
+
+// Type utilities for easier consumption
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface ApiRequestTypes {
+	// Request types will be available here as they are generated
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)